src/main/jhipster/src/main/webapp/app/ontology/variable-details/valid-values.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit } from '@angular/core';
+import { JhiLanguageService } from 'ng-jhipster';
+import { VariableDetails } from '../../shared/ontology/model/variable-details';
+import { VariableDetailsContext } from './variable-details.context';
+
+enum DataType {
+    NUMERIC = 'Numeric',
+    CATEGORICAL = 'Categorical',
+    CHARACTER = 'Character'
+}
```

Comment:
maybe we can move it to `/shared/ontology`

Reply:
Dont we have DATE data type? Just thinking that when creating a scale, Date is also available and for the germplasm details - attributes section we are requested to handle date data type
@lik-83

Reply:
I moved to '/shared/ontology'. Regarding the data types values, I just only added the ones that I have used in the popup.